src/App.js

```diff
@@ -1,19 +1,20 @@
 import React, { Suspense, lazy, useState } from "react";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import { useTranslation } from "react-i18next";
+import { Provider } from 'react-redux';
+import store from "./context/redux/store";
 import DrawerMenu from "./components/DrawerMenu";
 import Header from "./components/Header";
 import Footer from "./components/Footer";
 import ToastContext from "./context/ToastContext";
 import ToastNotification from "./components/Toast alerts/ToastNotification";
-import TransactionDetails from "./pages/transactions/TransactionDetails";
+import DrawerDetails from "./components/DrawerDetails";
 
 const Transactions = lazy(() => import("./pages/transactions/Transactions"));
```

Comment:
please use prefetching strategy for lazy loading with webpack comments (that strategy enables the prefetching of the files and caching them before they are requested on a lazy loaded route (the timing of fetching of a component from cache will be of 1-2ms)